conf/configuration.yml

```diff
@@ -1,3 +1,5 @@
+disabledAPIs: []
```

Comment:
why is this needed?

Reply:
For internal use cases, we require certain APIs, such as Org/Team-related or IAM-related APIs, to be disabled. This would include the regex patterns specifying the APIs that should not be accessible.